bin/lang-js/src/function.ts

```diff
@@ -91,12 +91,17 @@ export interface OutputLine {
   message: string;
 }
 
+// This is the default timeout, configured in minutes.
+const defaultTimeout = 60; // TODO(nick): make this configurable when deploying SI
+
 export async function executeFunction(kind: FunctionKind, request: Request) {
   // Run Before Functions
   const ctx = ctxFromRequest(request);
 
+  const timeout = request.timeout ?? defaultTimeout;
```

Comment:
Right now, this is configurable by request. We want that for two reasons: testing and tunable timeouts for ourselves. This PR does not cover deployment-level configuration and that will be covered in a second pass.